lib/schedule/health/checkers/trip_stop_times_checker.ex

```diff
@@ -19,7 +20,16 @@ defmodule Schedule.Health.Checkers.TripStopTimesChecker do
   @spec healthy_route?(timepoint_config()) :: boolean
   defp healthy_route?(%{route_id: route_id, min_length: min_length}) do
     with {:trip, %Trip{stop_times: stop_times}} <- trip(route_id) do
-      length(stop_times) >= min_length
+      length = length(stop_times)
+      passfail = length >= min_length
+
+      if(!passfail) do
+        Logger.warning(
+          "Trip Stop Times Checker failed on route #{route_id}. min_legth=#{min_length} length=#{length}"
```

Comment:
One thought - since it's checking stop times for a particular trip, it might be good to log the trip ID as well (you can just additionally pull that out of the `%Trip{...}` struct on line 22).